specs/identity.test.ts

```diff
@@ -3,12 +3,18 @@ import { identity } from '../mod.ts'
 import { eq } from './utils/utils.ts'
 
 describe('identity', () => {
-  it('should return its first argument', () => {
-    it('returns its first argument', function () {
-      eq(identity(undefined), undefined)
-      eq(identity('foo'), 'foo')
-      // @ts-ignore
-      eq(identity('foo', 'bar'), 'foo')
-    })
+
+  it('returns its first argument', function () {
+    eq(identity(undefined), undefined)
+    eq(identity('foo'), 'foo')
+    // @ts-ignore
+    eq(identity('foo', 'bar'), 'foo')
+  })
+
+  it('returns its return value of function', function () {
```

Comment:
* should return the return value of function being passed

Reply:
anything else